wasm/jit/engine.go

```diff
@@ -446,3 +447,182 @@ func (e *engine) builtinFunctionMemoryGrow(f *compiledWasmFunction) {
+		case *wazeroir.OperationGt:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationLe:
+			err = compiler.compileLe(o)
+		case *wazeroir.OperationGe:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationAdd:
+			err = compiler.compileAdd(o)
+		case *wazeroir.OperationSub:
+			err = compiler.compileSub(o)
+		case *wazeroir.OperationMul:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationClz:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationCtz:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationPopcnt:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationDiv:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationRem:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationAnd:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationOr:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationXor:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationShl:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationShr:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationRotl:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationRotr:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationAbs:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationNeg:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationCeil:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationFloor:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationTrunc:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationNearest:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationSqrt:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationMin:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationMax:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationCopysign:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationI32WrapFromI64:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationITruncFromF:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationFConvertFromI:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationF32DemoteFromF64:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationF64PromoteFromF32:
+		case *wazeroir.OperationI32ReinterpretFromF32,
+			*wazeroir.OperationI64ReinterpretFromF64,
+			*wazeroir.OperationF32ReinterpretFromI32,
+			*wazeroir.OperationF64ReinterpretFromI64:
+			err = fmt.Errorf("unsupported operation")
+		case *wazeroir.OperationExtend:
+			err = fmt.Errorf("unsupported operation")
+		}
+		if err != nil {
+			return nil, fmt.Errorf("failed to compile operation %s: %w", op.Kind().String(), err)
```

Comment:
e.g.: `failed to compile Wasm function: failed to compile operation Ge: unsupported operation`